packages/main/src/themes/sap_horizon/RatingIndicator-parameters.css

```diff
@@ -6,12 +6,15 @@
 	--_ui5_rating_indicator_readonly_item_height: 0.75em;
 	--_ui5_rating_indicator_readonly_item_width: 0.75em;
 	--_ui5_rating_indicator_readonly_item_spacing: 0.1875rem 0.1875rem;
+	--_ui5_rating_indicator_min_font_size: 24px;
 }
 
 [data-ui5-compact-size],
 .ui5-content-density-compact,
 .sapUiSizeCompact {
-	--_ui5_rating_indicator_readonly_item_height: 0.5em;
-	--_ui5_rating_indicator_readonly_item_width: 0.5em;
-	--_ui5_rating_indicator_readonly_item_spacing: 0.125rem 0.125rem;
+	--_ui5_rating_indicator_item_height: 1em;
+	--_ui5_rating_indicator_item_width: 1em;
+	--_ui5_rating_indicator_readonly_item_height: 0.75em;
```

Comment:
why do we modify the readonly sizes? they are not a subject of the topic as they are not clickable and do not require a minimum size touch area?

Reply:
In the base .css for RatingIndicator , for "Compact" mode we have a setting for the read-only icons to have size 0.67em. This is because of all themes that do not have parameter overrides - like Quartz for example. So if I don't override it for Horizon themes, the read-only icons will be shrinked to 0.67em when switching to "Comact" mode for Horizon, which is not by spec. The active icons will keep their size when switching from Cozy to Compact mode, but the read-only icons will change their size.